apps/aehttp/src/aehttp_api_encoder.erl

```diff
@@ -160,54 +160,58 @@ type2id(contract_pubkey) -> contract;
 type2id(name)            -> name;
 type2id(oracle_pubkey)   -> oracle.
 
-type2enc(key_block_hash)   -> ?BASE58;
-type2enc(micro_block_hash) -> ?BASE58;
-type2enc(block_pof_hash)   -> ?BASE58;
-type2enc(block_tx_hash)    -> ?BASE58;
-type2enc(block_state_hash) -> ?BASE58;
-type2enc(channel)          -> ?BASE58;
-type2enc(contract_pubkey)  -> ?BASE58;
-type2enc(contract_bytearray)-> ?BASE64;
-type2enc(transaction)      -> ?BASE64;
-type2enc(tx_hash)          -> ?BASE58;
-type2enc(oracle_pubkey)    -> ?BASE58;
-type2enc(oracle_query)     -> ?BASE64;
-type2enc(oracle_query_id)  -> ?BASE58;
-type2enc(oracle_response)  -> ?BASE64;
-type2enc(account_pubkey)   -> ?BASE58;
-type2enc(signature)        -> ?BASE58;
-type2enc(commitment)       -> ?BASE58;
-type2enc(peer_pubkey)      -> ?BASE58;
-type2enc(name)             -> ?BASE58;
-type2enc(state)            -> ?BASE64;
-type2enc(poi)              -> ?BASE64;
-type2enc(state_trees)      -> ?BASE64;
-type2enc(call_state_tree)  -> ?BASE64.
+type2enc(key_block_hash)       -> ?BASE58;
+type2enc(micro_block_hash)     -> ?BASE58;
+type2enc(block_pof_hash)       -> ?BASE58;
+type2enc(block_tx_hash)        -> ?BASE58;
+type2enc(block_state_hash)     -> ?BASE58;
+type2enc(channel)              -> ?BASE58;
+type2enc(contract_pubkey)      -> ?BASE58;
+type2enc(contract_bytearray)   -> ?BASE64;
+type2enc(contract_store_key)   -> ?BASE64;
```

Comment:
May I kindly ask you refresh [the tests in `aehttp_api_encoder_tests` for the contract-related encodings](https://github.com/aeternity/aeternity/blob/f5c46b80561697c2c41d03f9cc2dc7400b153e9e/apps/aehttp/test/aehttp_api_encoder_tests.erl#L10-L26)?